src/components/AnonymousChat/MessageChannelItem.tsx

```diff
@@ -77,6 +78,9 @@ const MessageChannelItem: (props: MessageChannelItemProps) => React.ReactElement
       channelType={item?.channelType}
       dbAnonUserInfo={dbAnonUserInfo}
       hasAttachment={item?.rawJson?.message?.attachments?.length > 0}
+      containerBackgroundColor={
+        item?.channelType === 'PM' ? COLORS.signedPmGroupChannel : COLORS.anonPmGroupChannel
+      }
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The logic for setting the `containerBackgroundColor` based on the `channelType` looks correct. However, it's not clear what 'PM' stands for in this context. If possible, consider using more descriptive constants for channel types to improve readability.
```diff
- item?.channelType === 'PM' ? COLORS.signedPmGroupChannel : COLORS.anonPmGroupChannel
+ item?.channelType === CHANNEL_TYPES.PRIVATE_MESSAGE ? COLORS.signedPmGroupChannel : COLORS.anonPmGroupChannel
```
In the above suggestion, `CHANNEL_TYPES` could be an object that maps more descriptive names to the actual values used in your application. This would make the code easier to understand for other developers.


<!-- This is an auto-generated comment by OSS CodeRabbit -->